packages/table/src/table.tsx

```diff
@@ -410,6 +410,11 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
     private rowHeaderElement: HTMLElement;
     private scrollContainerElement: HTMLElement;
 
+    // when this variable is true, we'll need to synchronize quadrant views
+    // after the update. this variable lets us avoid expensively diff'ing
+    // columnWidths and rowHeights on each update.
+    private didColumnOrRowSizesChangeInLatestUpdate: boolean = false;
```

Comment:
wont this member be typed as `boolean` automatically if you set it to false?

Reply:
Is there... anything better that you can name this? I can't think of anything, but, while I agree with the usage, the name is pretty awful.

Reply:
Changed to `didUpdateColumnOrRowSizes`.